src/main/java/core/basesyntax/Main.java

```diff
@@ -1,7 +1,7 @@
-package core.basesyntax;
+package main.java.core.basesyntax;
```

Comment:
there is no need in specifying `man.java` it should be ok without it, same for all other classes